hw/ip/i2c/data/i2c.prj.hjson

```diff
@@ -12,10 +12,10 @@
       {
         version:            "0.5",
         life_stage:         "L1",
-        design_stage:       "D1",
+        design_stage:       "D2",
         verification_stage: "V1",
         dif_stage:          "S0",
-        commit_id:          "ff9c06167bc34d921974839017dec54f0fb8347a",
+        commit_id:          "a44bdf910e4941a40a2bccfbecb525ac0b68f727",
```

Comment:
The commit ID is not needed as long as the design is still under development.
It only needs to be added when we roll back design stages (e.g., go back from D2 to D1) to keep the latest snapshot of the previous design.
Hence, this line can be removed.

Reply:
This line is removed.